### PR TITLE
created ui for managing tenders

### PR DESCRIPTION
**Pull request by Sunday-Binakin**
- Added submission forms for new tenders
- Developed interfaces for tender review
- Implemented tracking views for tender statuses and history
- Integrated PDF preview, view, and delete functionalities
- Ensured responsive design for various screen sizes
- Included validation and error handling for form inputs

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,18 +1,33 @@
 import React from "react";
-import "./App.css";
+import { ThemeProvider, createTheme } from '@mui/material/styles';
+import CssBaseline from '@mui/material/CssBaseline';
```
Comment (samuelBright09) [inline src/App.jsx:3]: what are these packages I see you've installed. What are their purposes.

Comment (samuelBright09) [inline src/App.jsx:3]: what is CSS baseline goin to be used for

**Review by samuelBright09 — COMMENTED** [review]
Before you started work, you didnt have the updated main, with roberta's code ryt?

**Review by samuelBright09 — COMMENTED** [review]
I dont understand what you're doing in the App.jsx file. I dont know why you decided to clear the app.css and also the decided to put the router = createBrowserRoute{} above the function App (). I'm sorry but I wouldn't merge your work cos even if I resolve the conflict you've caused, there's still going to be confusion in our code cos you're out of our scope. the packages you've installed and some other codes you've written is not in our scope. pls if youy can rectify that. pls do and sync your changes... If not, I would be forced to close your PR

File: src/App.jsx (hunk 1 above)
Comment (Sunday-Binakin) [reply src/App.jsx:3]: component library

Comment (Sunday-Binakin) [reply src/App.jsx:3]: it comes with the component library

File: src/App.jsx [hunk 2]
```diff
@@ -1,18 +1,33 @@
 import React from "react";
-import "./App.css";
```
[the remark this answers is not in the document]
Comment (Sunday-Binakin) [reply src/App.jsx:2]: should be an oversight , cuz i don't have any business there

**Comment by Sunday-Binakin** [comment]
> I dont understand what you're doing in the App.jsx file. I dont know why you decided to clear the app.css and also the decided to put the router = createBrowserRoute{} above the function App (). I'm sorry but I wouldn't merge your work cos even if I resolve the conflict you've caused, there's still going to be confusion in our code cos you're out of our scope. the packages you've installed and some other codes you've written is not in our scope. pls if youy can rectify that. pls do and sync your changes... If not, I would be forced to close your PR

about the conflict. I am using a package that needs to be imported at the app.jsx. besides I told you I could merge the work you said no, that is your work. so if you have a hard time merging the work, all you need to do is reach out

**Review by samuelBright09 — COMMENTED** [review]
I have done best to resolve the conflicts. If theres any explanation that they would need on the packages that you installed be sure to be ready to give the team a comprehensive answer. Thanks for being patient with the situation.

**Review by samuelBright09 — APPROVED** [review]

**Pull request merged**